### ISSUE TITLE
Autorestart and Virus issue

### ISSUE THREAD

**Issue by TrueLogiko** [AppsForDesktop/Facebook-desktop#1]
PC gone AutoRestart after installation.
3 security vendors from Virustotal flagged installer as malicious.
Any coder checked this app by himself?

**Comment by AppsForDesktop** [comment]
@TrueLogiko, please provide more information what happened, your Windows version and also please tell us if software did installed and if it works.
Please provide steps you performed, so we can pinpoint the issue, as it is very strange that PC restarted, as the code does not contain anything that can do that.

**Comment by TrueLogiko** [comment]
@AppsForDesktop , The program was installed and is running correctly as it should be. AutoRestart happened while usual installation, instead of showing me the finish button, Just after installation bar met approx 100%.
Currently running Windows 10 Pro  Version 10.0.19045.3324

Here are virus checking results. 
https://virusscan.jotti.org/en-US/filescanjob/iqppsdd2ev
https://www.virustotal.com/gui/file/be5fc719ec4727d7b5f1e2abf84e9a633ee59c3c0a85f8ee921fd5c19480a784
Hope they are false-positive because i am using this app now)))

Please tell me how to configure program settings, maybe i should change some cfg file somewhere? And  where are program/browser cache, history, cookies, history and other data located. I would wan sometime to delete them and "refresh" the program.
Thanks

**Comment by AppsForDesktop** [comment]
@TrueLogiko, Thank you for the provided information. But we can not reproduce the issue with AutoRestart, so we will assume that it happened because of every Windows system is unique(different AVs and configuration), and this is unique case.
Yes, some VirusTotal AVs detect our program as threat we assume this is because or software is not signed by Microsoft.
The program has no configuration.
The cache is stored at "C:\Users\UserName\AppData\Local\Temp\Facebook", you can check this at this [code line](https://github.com/AppsForDesktop/Facebook-desktop/blob/33f159366e03b5a26fb0ce540d61a4865a42634d/Form1.cs#L37C66-L37C76). To clear cache, just delete whole "Facebook" directory

**Comment by qqtesthacker** [comment]
Agreed with the virus detected.Besides, all of the multiple apps cannot load the video inside.Do I need to install some other plugin?

**Comment by qqtesthacker** [comment]
This is my framework installed![image](https://github.com/AppsForDesktop/Facebook-desktop/assets/20235125/82c3ee16-cc65-45d8-8792-f1d8ca111db1)
